audb/core/load.py

```diff
@@ -79,18 +79,19 @@ def _cached_files(
         file_version = LooseVersion(deps.version(file))
         for cache_version, cache_root, cache_deps in cached_versions:
             if cache_version >= file_version:
-                if deps.checksum(file) == cache_deps.checksum(file):
-                    path = os.path.join(cache_root, file)
-                    if flavor.format is not None:
-                        path = audeer.replace_file_extension(
-                            path,
-                            flavor.format,
-                        )
-                    if os.path.exists(path):
-                        found = True
-                        break
+                if file in cache_deps:
+                    if deps.checksum(file) == cache_deps.checksum(file):
+                        path = os.path.join(cache_root, file)
+                        if flavor and flavor.format is not None:
+                            path = audeer.replace_file_extension(
+                                path,
+                                flavor.format,
+                            )
+                        if os.path.exists(path):
+                            found = True
+                            break
         if found:
-            if flavor.format is not None:
+            if flavor and flavor.format is not None:
```

Comment:
You are stating:

> do not replace extension when looking for tables

Do you mean this line? What has the `flavor.format` todo with looking for tables?

Reply:
We are using the same function for finding tables and media.

Reply:
Ah, ok, so adding `if flavor` avoids applying it to a table, correct?

Reply:
Yes, since we now pass `flavor=None` when we look for tables.